### ISSUE TITLE
integration with TensorFlow

### ISSUE THREAD

**Issue by edgimar** [reiinakano/xcessiv#38]
While the current interface provided by xcessiv is, well excessively nice, I am hoping that it will be possible to integrate TensorFlow models / estimators with it.  Has this been done, or would it be possible to add any documentation with a basic example?

**Comment by reiinakano** [comment]
Hey @edgimar, the requirement for an xcessiv estimator is that it has fit, get_params, and set_params. Not sure about tensorflow, but Keras has a class KerasClassifier and KerasRegressor that works properly with this. I've been planning to add documentation showing how to do this.

Only drawback is that training sets have to be in memory because of the (X, y) scikit learn paradigm. I'm not sure how to get around that without breaking everything else.

**Comment by edgimar** [comment]
Yeah, that was my next question:  how to work with multiple training-set batches.  sklearn also has the partial_fit API method which is made to handle this, but it sound like xcessiv isn't set up to make use of this?

**Comment by reiinakano** [comment]
Sadly, no. I'll try to see if I can come up with a way that makes it work, but for now, don't expect anything too soon.
